File: Python/Selection Sort.py
list=[8,9,5,7,6,1,3,2,4]

length = len(list)

for i in range(length-1):
    indexMin = i
    for j in range(i + 1, length):
        if list[indexMin] > list[j]:
            indexMin = j
    temp = list[indexMin]
    list[indexMin] = list[i]
    list[i] = temp
    print(list)

File: C/Selection Sort.c
#include <stido.h>

int main()
{
    int list[] ={8,9,5,7,6,1,3,2,4};
    int i,j,temp;
    int indexMin;

    for(i=0; i<sizeof(list)/sizeof(int)-1;i++)
    {
        indexMin = 1;
        for(j=i+1; j < sizeof(list)/sizeof(int); j++)
        {
            if(list[indexMin] > list[j])
                indexMin = j;

            temp = list[indexMin];
            list[indexMin] = list[i];
            list[i] = temp;
        }
    }

}
